### PR TITLE
Rent routes implementation

### PR DESCRIPTION
**Pull request by marcegl91**
### Summary

- Add rent#index, rent#show and rent#create

### Trello Card

https://trello.com/c/uqmzkgKX/28-crear-endpoint-de-creacion-de-rent-e-index

### CODE REVIEW

File: wbooks-api/app/controllers/api/v1/rents_controller.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+      def show
+        rent = Rent.find(params[:id])
+        render json: rent, serializer: RentSerializer
+      end
+
+      def create
+        rent = Rent.new(rent_params)
+        if rent.save
+          head :ok
```
Comment (facucompe) [inline wbooks-api/app/controllers/api/v1/rents_controller.rb]: We usually return the created object in POST requests

File: wbooks-api/app/serializers/rent_serializer.rb [hunk 2]
```diff
@@ -1,6 +1,4 @@
 class RentSerializer < ActiveModel::Serializer
-  attributes :id, :book_id, :user_id, :returned_at
-
-  attribute :rented_from, key: :from
-  attribute :rented_to, key: :to
+  attributes :id, :user_id, :book_id, :rented_from, :rented_to, :returned_at, :created_at
+  attributes :updated_at
```
Comment (facucompe) [inline wbooks-api/app/serializers/rent_serializer.rb]: Why are you doing this? Can you add it to the other attributes list

File: wbooks-api/app/controllers/api/v1/rents_controller.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
+module Api
+  module V1
+    class RentsController < ApiController
+      def index
+        rents = Rent.all
+        render_paginated rents, each_serializer: RentSerializer
```
Comment (floresv) [inline wbooks-api/app/controllers/api/v1/rents_controller.rb]: same line, fix with equal to PR 8.

File: wbooks-api/app/controllers/api/v1/rents_controller.rb [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      end
+
+      def show
+        rent = Rent.find(params[:id])
+        render json: rent, serializer: RentSerializer
```
Comment (floresv) [inline wbooks-api/app/controllers/api/v1/rents_controller.rb]: Same line.. :thinking:

File: wbooks-api/app/controllers/api/v1/rents_controller.rb [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+      end
+
+      def create
+        rent = Rent.new(rent_params)
```
Comment (floresv) [inline wbooks-api/app/controllers/api/v1/rents_controller.rb:15]: I prefer use https://apidock.com/rails/ActiveRecord/AttributeAssignment/assign_attributes

File: wbooks-api/app/controllers/api/v1/rents_controller.rb (hunk 1 above)
Comment (floresv) [reply wbooks-api/app/controllers/api/v1/rents_controller.rb]: I agree, and use https://quickrails.com/ruby-one-line-if-else-statement/

By the way, I agree with this https://en.wikipedia.org/wiki/Fail-fast

Return Ok at the end.

File: wbooks-api/app/serializers/rent_serializer.rb (hunk 2 above)
Comment (floresv) [reply wbooks-api/app/serializers/rent_serializer.rb]: I agree with facu

**Review by floresv — CHANGES_REQUESTED** [review]
And the controllers testing? :thinking:

File: wbooks-api/app/controllers/api/v1/rents_controller.rb (hunk 5 above)
Comment (floresv) [reply wbooks-api/app/controllers/api/v1/rents_controller.rb:15]: forget it

**Review by floresv — APPROVED** [review]
LGTM

**Review by floresv — APPROVED** [review]

**Review by facucompe — APPROVED** [review]

**Pull request merged**